docs/containers-storage.conf.5.md

```diff
@@ -148,31 +126,70 @@ Sets the log level of devicemapper.
     7: LogLevelDebug
 
 **min_free_space**=""
-
-Specifies the min free space percent in a thin pool required for new device creation to succeed. Valid values are from 0% - 99%. Value 0% disables. (default: 10%)
+  Specifies the min free space percent in a thin pool required for new device creation to succeed. Valid values are from 0% - 99%. Value 0% disables. (default: 10%)
 
 **mkfsarg**=""
+  Specifies extra mkfs arguments to be used when creating the base device.
 
-Specifies extra mkfs arguments to be used when creating the base device.
+**mountopt**=""
+  Comma separated list of default options to be used to mount container images.  Suggested value "nodev".
 
-**use_deferred_deletion**=""
+**size**=""
+  Maximum size of a container image.   This flag can be used to set quota on the size of container images.
```

Comment:
What size?  Gb, Mb, Kb, Bytes?  Can you specify 120M for 120MB say?

Reply:
Fixed